src/features/Posts/PostsList/PostsList.js

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import { Alert } from 'reactstrap';
 import styled from 'styled-components';
 import { List, LoadingIndicator } from 'components';
-import Post from './Post';
+import Post from './PostView';
 
-const PostsList = ({ posts, isLoading }) =>
+const PostsList = ({ posts, isLoading, deletePost }) =>
   <List
```

Comment:
Please add deletePost to List properties and pass all props `children(props)` in List component

Reply:
What do you exactly mean? Could you specify with example on ```children(props)```

Reply:
Look at List.js file